### DISCUSSION TITLE
I can't build a project on Unity

### DISCUSSION THREAD

**Discussion by juiceNewton-1** [codemagic-ci-cd/codemagic-docs#2337] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Unity

### Steps to reproduce

I have the following script in the codemagic.yaml file: 

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/116092676/fb536b4f-c463-491d-be27-d1aa5595f695)

as well as unity_credentials with an active license:

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/116092676/70f6e12a-7fef-4485-8763-d683614a853a)
 

### Expected results

We need to make a build of the project and send its review to the AppStore

### Actual results

When trying to build a project, I get the following:

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/116092676/8f6cb9b0-2861-4db2-a449-3bae0f53312e)

**Comment by mikhail-tokarev** [comment]
looks like unity version is missed in `environment` section. You can read more here https://docs.codemagic.io/knowledge-others/install-unity-version/

**Comment by icarusdust** [comment]
Closing the thread due to inactivity. Let us know in case of any further questions by replying to this thread or by opening a new one.
